Transport/Simple.H

```diff
@@ -379,6 +400,7 @@ struct SimpleTransport
     amrex::Array4<const amrex::Real> const& T_in,
     amrex::Array4<const amrex::Real> const& Rho_in,
     amrex::Array4<amrex::Real> const& D_out,
+    amrex::Array4<amrex::Real> const& chi_out,
```

Comment:
So this changes the call signature of `get_transport_coeffs`. I think it would be necessary to update the other transport types to have the same signature that way the call site doesn't take an if condition.

Reply:
Added the appropriate call signatures to constant and sutherland transport types

Reply:
yes, perfect.